go.mod

```diff
@@ -7,3 +7,5 @@ require (
 	github.com/go-openapi/swag v0.18.0
 	github.com/go-openapi/validate v0.18.0
 )
+
+go 1.13
```

Comment:
Should remove this file...not sure whether we should be strict to 1.13. Should allow older golang (1.12 for example).

Reply:
This got merged in by mistake. I'll remove this.